backend/models/user.go

```diff
@@ -20,6 +20,16 @@ type User struct {
 	Debt              float64   `json:"debt"`
 }
 
+// SSHKey represents an SSH key for a user
+type SSHKey struct {
```

Comment:
make name and public key unique as i was able to add the same key and name for user multiple times